client/src/components/shared/fallback/ErrorFallback.tsx

```
@@ -0,0 +1,11 @@
+import type { FallbackProps } from "react-error-boundary";
+
+export const ErrorFallback = ({ error, resetErrorBoundary }: FallbackProps) => {
+  return (
+    <div role="alert" style={{ color: "red", padding: "1rem" }}>
+      <p>Something went wrong:</p>
+      <pre style={{ whiteSpace: "pre-wrap" }}>{error.message}</pre>
+      <button onClick={resetErrorBoundary}>Try again</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)